styles.css

```diff
@@ -0,0 +1,6 @@
+img {
```

Comment:
i suggest you double check that all your tag line names are correct in your styles.css as it is in your index.html file so the two files can work with each other to produce a result